tools/build/Makefile.rules

```diff
@@ -212,7 +212,9 @@ inc-subdirs = \
 			$(call parse-sample,$(sample),$(subdir)) \
 		) \
 		$(clean-control) \
-	)
+	) \
+	$(eval requires-private := $(sort $(requires-private-y) $(requires-private-m))) \
```

Comment:
only `requires-private-y` is needed. Whatever goes as shared modules/objects doesn't count in libsoletta.so and thus not in `soletta.pc`